src/Nelmio/Alice/Persister/Doctrine.php

```diff
@@ -55,4 +60,21 @@ public function find($class, $id)
 
         return $entity;
     }
+
+    private function getPersistableClasses()
```

Comment:
I'm not sure about what you mean here: persistable classes are already stored in a property after initialization.
Do you suggest to move the initialization in the constructor and use the property in `persist` without additional method call ?